pkgs/applications/misc/1password-gui/default.nix

```diff
@@ -1,77 +1,105 @@
-{ lib
-, stdenv
-, fetchurl
+{ fetchurl
 , makeWrapper
-, electron_11
-, openssl
+, alsaLib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, gdk-pixbuf
+, glib
+, gtk3
+, libdrm
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, systemd
+, libX11
+, libxcb
+, libXcomposite
+, libXdamage
+, libXext
+, libXfixes
+, libXrandr
+, libxshmfence
+, lib
```

Comment:
Yes indeed, much better!

One final nit: `lib` should always be the first input.

The rest LGTM :)